Resources/input/fsh/ISiKPatient.fsh

```diff
@@ -42,6 +42,8 @@ Description: "Dieses Profil beschreibt die Nutzung von administrativen Patienten
     * identifier.value MS
     * display MS
 * active MS
+  * ^definition = "Einschränkung der Übergreifenden MS-Definition: Verfügt ein bestätigungsrelevantes System nicht über die Datenstruktur zur Hinterlegung des Aktivitätsstatus einer Pateinten-Ressource, so MUSS dieses System die Information NICHT in der Observation abbilden. Das System SOLL jedoch den Aktivitätsstatus hart kodieren in der Patienteninstanz (Patient.active auf 'true').
```

Comment:
typo: Pateinten --> Patienten (suggestion: Spell-Check in der IDE überprüfen)
issue: Observation --> Patient-Ressource
question: das Feld hat Kardinalität 0..1. warum müssen Systeme ohne Unterstützung hier etwas "hart kodieren" und nicht einfach das Element weglassen?

Reply:
Die Information ist wichtig um als Client nicht aus versehen mit einer stornierten Patientenressource zu arbeiten. 1..1 wäre hier sinnvoll.